.eslintrc.js

```diff
@@ -27,6 +27,8 @@ module.exports = {
     // Allow writing components as arrow functions.
     'react/function-component-definition': 'off',
     'react/prop-types': 'off',
+    'react/jsx-props-no-spreading': 'off',
+    'react/react-in-jsx-scope': 'off',
```

Comment:
We don't need React to be in scope with the "new JSX transform" from React 17.
https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html